index.html

```diff
@@ -123,97 +117,75 @@
     <dt>`entryType`</dt>
     <dd>This attribute MUST return the DOMString "`server`".</dd>
     <dt>`startTime`</dt>
-    <dd>This attribute MUST return value 0.</dd>
+    <dd>This attribute MUST return value 0. Because there can be no guarantee of clock synchronization between client, server, and intermediaries, it is impossible to map a meaningful `startTime` onto the clients timeline.</dd>
     <dt>`duration`</dt>
     <dd>
-      This attribute MUST return a <a data-cite="hr-time-2#idl-def-domhighrestimestamp">`DOMHighResTimeStamp`</a> [[!HR-TIME-2]] that contains the server-specified metric value, or value 0.
+      This attribute MUST return a number that contains the server-specified metric value, or value 0.
     </dd>
   </dl>
-</section>
-<section data-dfn-for="ServerEntry">
-  ## <dfn>ServerEntry</dfn> interface
-  
-  <pre class='idl'>
-  interface ServerEntry : PerformanceEntry {
-    readonly attribute DOMString metric;
-    readonly attribute DOMString description;
-    serializer = { inherit, attribute };
-  };
-  </pre>
-  
-  <dl>
-    <dt><dfn>metric</dfn> attribute</dt>
-    <dd>This attribute MUST return the server-specified metric name.</dd>
-    <dt><dfn>description</dfn> attribute</dt>
-    <dd>This attribute MUST return the server-specified metric description, or an empty string.</dd>
-    <dt><dfn>serializer</dfn></dt>
-    <dd>See [[!WebIDL]] for serializer behavior of <a data-cte="WebIDL-LS#inherit">`inherit`</a> and <a data-cte="WebIDL-LS#attribute">`attribute`</a>.</dd>
-  </dl>
-</section>
 
-<section data-dfn-for="Performance">
-  ## Extensions to `Performance` Interface
-  
-  The <code><dfn data-cite="hr-time-2#the-performance-interface">Performance</dfn></code> interface is extended with the following methods and attribute:
-
-  <pre class="idl">
-  partial interface Performance {
-    void clearServerTimings();
-    void setServerTimingBufferSize(unsigned long maxSize);
-    attribute EventHandler onservertimingbufferfull;
-  };
+  <pre class='idl'>
+    [Exposed=(Window,Worker)]
+    interface PerformanceServerTiming : PerformanceEntry {
```

Comment:
I'm not sure that inheriting from PerformanceEntry here is the right way to go:
* `name` is entirely redundant as it would be identical to PerformanceResourceTiming's `name`
* `entryType` is not necessary, as we'll no longer have a way to access server timing entries directly
* `startTime` is 0
* `duration` is the only meaningful value, and one could claim that its naming is not ideal in the ServerTiming context. (also we're implicitly converting a number to a `DOMHighResTimestamp`)

Maybe we should avoid the inheritance and just add a `value` attribute here?